src/java.base/unix/native/libjsig/jsig.c

```diff
@@ -42,14 +42,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-
-#if (__STDC_VERSION__ >= 199901L)
```

Comment:
Since this does include stdbool.h already, this change looks ok

Reply:
Okey.